src/main/java/ai/zuva/example/Example.java

```diff
@@ -0,0 +1,106 @@
+package ai.zuva.example;
+
+import ai.zuva.classification.ClassificationRequest;
+import ai.zuva.extraction.ExtractionRequest;
+import ai.zuva.fields.Field;
+import ai.zuva.files.ZdaiFile;
+import ai.zuva.api.ZdaiApiClient;
+import ai.zuva.language.LanguageRequest;
+import ai.zuva.ocr.OcrRequest;
+import ai.zuva.extraction.ExtractionData;
+import ai.zuva.extraction.ExtractionResults;
+import ai.zuva.fields.FieldMetadata;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+
+public class Example {
+    public static void main(String[] args) throws Exception {
+        String token = System.getenv("ZUVA_TOKEN");
+        if (token == null) {
+            throw(new Exception("Environment variable ZUVA_TOKEN not set"));
+        }
+
+        String url = System.getenv("ZUVA_URL");
+        if (url == null) {
+            throw(new Exception("Environment variable ZUVA_URL not set"));
+        }
+
+        String documentPath = System.getenv("ZUVA_DEMO_DOC");
+        if (documentPath == null) {
+            // default to looking for the demo file in the current working directory
+            documentPath = "CANADAGOOS-F1Securiti-2152017.PDF2";
+        }
+
+        ZdaiApiClient client = new ZdaiApiClient(url, token);
+
+        ZdaiFile zdaiFile = ZdaiFile.submitFile(client, new File(documentPath));
+        System.out.println(String.format("Uploaded file with id %s expires at %s", zdaiFile.fileId, zdaiFile.expiration));
+
+        System.out.println("\nClassifying Document type:");
+        ClassificationRequest classificationRequest = ClassificationRequest.createClassificationRequest(client, zdaiFile);
```

Comment:
What do you think about simplifying the name of this function as well so that it stutters less? 
Something like `ClassificationRequest.createRequest(client, zdaiFile)` 
or maybe even just `ClassificationRequest.create(client, zdaiFile)`

This would apply for the other request classes as well.

Reply:
I went with `createRequest` for now. I also wondered whether all of the `getStatus`, `getResults` etc. methods should  use the verb `fetch` or something instead. Right now they kind of look like they're just simple getters for private attributes, not actual API calls.

Reply:
I actually had a similar thought regarding the create functions. At first I wondered if it was simply creating an instance of the request class. I was going to suggest something like `submitRequest`  instead, but then I realized you were using the same wording as the Postman collection so at least that was consistent.